js/script.js

```diff
@@ -3,15 +3,16 @@
 let userSeq = [];
 let compSeq = [];
 let strictMode = false;
-let power = false;
-let sound = true;
+let powerSelection = false;
+let sound = false;
 let winTheGame;
-let userTurn;
+let loseTheGame;
+let round = 1;
 let intervalId;
-let buttonFlash;
+let quadFlash;
```

Comment:
do we need all these global variables? we shouldn't be keep many global variables especially when we don't need them/can do without them.